packages/react-native/postInitScript.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import chalk from 'chalk';
+import {logger} from '@react-native-community/cli-tools';
+
+function printInitScript(
+  projectName: string,
+) {
+  const relativeXcodeProjectPath = `visionos/${projectName}.xcworkspace`;
```

Comment:
Are you sure this path will exist?

We cannot assume this path because Cocoapods are not installed 😅